data/jibauni/rebel jibauni 0 consultations.txt

```diff
@@ -0,0 +1,359 @@
+				flee
+			label "met and declined the jibauni"
+			`Upon landing, a Rebel comes up to you telling you to go to <planet> to consult with the leaders. You don't have another option, and so you tell the Jibauni that you will. As the Rebels seem to expect you to leave immediately, you decide to do so.`
+				launch
+			label flee
+			action
+				"reputation: Rebel Jibauni" = -1
+				set "jibauni: disliked by the rebels"
+	on complete
+		event "rebel jibauni ready first contact" 1
+		"reputation: Jibauni (Rebel)" = 10
+		conversation
+			`A Rebel Jibauni tells you to take off and re-land so that they can prepare for your arrival. You are then unceremoniously launched with some strange mechanism.`
+				launch
+
+event "rebel jibauni ready first contact"
+
+mission "Rebel Jibauni: Landed on Andwerfen"
+	landing
+	source Andwerfen
+	to offer
+		or
+			has "First Contact: Rebel Jibauni: declined"
+			has "First Contact: Rebel Jibauni: done"
+		not "attempted to shoot a rebel jibauni"
+		has "event: rebel jibauni ready first contact"
+	on offer
+		conversation
+			`A long wait ensues after you land on <origin>. The Rebel Jibauni seem to be trying to find someone you can speak your language. Feeling bored, you wonder about going to see what's happening.`
+			choice
+				`	(Hey, it's better then staying here the rest of the day.)`
+				`	(You have no idea what this world is about, and don't want to wander unbidden.)`
+					goto stay
+			`	As you wander around the spaceport of <origin>, you notice the unhappy residents make way, even in the thickest crowds, without being asked. At last, you run into a Jibauni who has something around its neck. It gives an exclamation of surprise, and then asks in a language that you can easily understand, "Why have you not come before? I will take you to the government building." It drags you through the city, and then down a side alley. After a long climp up a ramp, you enter a room which has a congregation of dinosaurs.`
+				goto conversation
+			label stay
+			`	After a few more hours, a Rebel comes rushing up, full of apologies. "I am so sorry. The leaders had different things to do, and the person supposed to find you was ambushed. Come with me now though for a consultation." It pulls you, with remarkable speed, through the streets of the city. Finally, after ducking down a side alley and going up a wide stairway, you are led into a room with a congregation of dinosaurs in it.`
+			label conversation
+			`	One of them begins to speak. "Welcome, fair stranger. How have you arrived here, and where have you come from?"`
+			choice
+				`	"I come from human space, of course."`
+					goto human
+				`	"Well, that may be hard to say."`
+					goto cantsay
+				`	"I came from your foolish originals the Jibauni. For some reason they are prejudiced about you and don't like the sound of the word 'Rebel'."`
```

Comment:
Originals feels off here. Not sure on the best replacement though... predecessors? Oppressors?

```suggestion
				`	"I came from your foolish oppressors, the Jibauni. For some reason, they are prejudiced against you and don't like the sound of the word 'Rebel'."`
```